test_conformance/api/test_kernel_private_memory_size.cpp

```diff
@@ -0,0 +1,27 @@
+#include "harness/errorHelpers.h"
+#include "harness/typeWrappers.h"
+#include <iostream>
+#include "procs.h"
+
+int test_kernel_private_memory_size(cl_device_id deviceID, cl_context context,
+                                    cl_command_queue queue, int num_elements)
+{
+    const char* TEST_KERNEL =
+        R"(__kernel void private_memory( __global ulong *buffer ){
+         volatile __private ulong x[1];
+         buffer[0] = x[0];
+         })";
+
+    clProgramWrapper program;
+    clKernelWrapper kernel;
+    cl_int err = create_single_kernel_helper(context, &program, &kernel, 1,
+                                             &kernels[i], "private_memory");
```

Comment:
Missing an error check here, e.g. `test_error(err, "create_single_kernel_helper");`

Reply:
Hi Ben, thank you so much for the comments.
I have added line you suggested and everything should be ready now.